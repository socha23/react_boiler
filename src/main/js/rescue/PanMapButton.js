import React from 'react'

const PanMapButton = ({onClick}) => <a style={{fontSize: 25}} onClick={e => {e.stopPropagation(); onClick(e)}}>
    <i className="glyphicon glyphicon-map-marker" />

</a>;

export default PanMapButton;