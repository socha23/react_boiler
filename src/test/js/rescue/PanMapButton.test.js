import React from 'react'
import {testSnapshot} from '../testUtils'

import PanMapButton from 'rescue/PanMapButton'

it('renders correctly', testSnapshot(
            <PanMapButton/>
));
