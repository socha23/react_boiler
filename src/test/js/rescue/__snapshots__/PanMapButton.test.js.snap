// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<PanMapButton>
  <a>
    <i
      className="glyphicon glyphicon-map-marker"
    />
  </a>
</PanMapButton>
`;
